In [2]:
#ファイルの読み込み
f = open("popular-names.txt","r")

#一行ずつ読み込んで、読み込めなくなるまで数える
count = 0
while True:
    readed_line = f.readline()
    if readed_line == "":
        break
    count += 1

print(count)
f.close()
#closeし忘れると、リソースを食ったり、書き込みが正常に行われないなどが起こる

2780


--UNIXコマンドによる確認画面--

(.venv1) shota@LAPTOP-7H1S837B:~/py_venvs/nlp100/100knock2024/shota/chapter02$ wc -l popular-names.txt
2780 popular-names.txt